pkg/catalog/versions/k8s.go

```diff
@@ -15,45 +17,45 @@ type KubernetesVersions struct {
 }
 
 var kubernetesVersions = map[string]KubernetesVersions{
-	"1.26.6": {
-		Kubernetes: "1.26.6",
+	"1.26.0": {
```

Comment:
This are part of a testing change that should have never been committed.  Don't merge until this is undone.